src/script/tracking/Countly.helpers.ts

```diff
@@ -17,9 +17,17 @@
  *
  */
 
+import {getWebEnvironment} from 'Util/Environment';
+
 import {Config} from '../Config';
 
 export function isCountlyEnabledAtCurrentEnvironment(): boolean {
+  const {isDev, isEdge, isInternal, isLocalhost, isStaging} = getWebEnvironment();
+
+  if (isDev || isEdge || isInternal || isLocalhost || isStaging) {
+    return true;
+  }
```

Comment:
```suggestion
  const {isProduction, isLinked} = getWebEnvironment();

  if (!isProduction &&  !isLinked) {
    return true;
  }
```
IMO, the opposite conditions are less.

Reply:
yeah but i would like to have them explicit named for readability :)